src/panels/KeybindsPanel/FullKeyboard/FullKeyboard.css

```diff
@@ -0,0 +1,86 @@
+/* Background */
+.simple-keyboard-background {
+  color: black;
+  font-size: 0.8em;
+  background-color: var(--mantine-color-dark-6);
+  border-bottom: 1px solid var(--mantine-color-dark-2);
+  border-radius: 5px;
```

Comment:
```suggestion
  border-radius: 5px;
  width: fit-content;
```

To prevent the div to stretch out indefinitely, like it does now: 
![image](https://github.com/user-attachments/assets/5d5839f8-bd94-4f29-9a2c-b9810226f67b)